frontend/src/scenes/settings/project/SessionRecordingSettings.tsx

```diff
@@ -409,3 +381,189 @@ export function ReplayCostControl(): JSX.Element | null {
+                            properties selected here.
+                        </p>
+                        <PropertySelect
+                            taxonomicFilterGroup={TaxonomicFilterGroupType.EventProperties}
+                            sortable={false}
+                            onChange={(properties: string[]) => {
+                                updateSummaryConfig({
+                                    ...currentConfig,
+                                    included_event_properties: properties,
+                                })
+                            }}
+                            selectedProperties={currentConfig.included_event_properties || []}
+                            addText="Add property"
+                        />
+                    </div>
+                    <div>
+                        <h3 className="flex items-center gap-2">
+                            <IconSelectEvents className="text-lg" />
+                            Important user properties
+                        </h3>
+                        <p>
+                            We always send the first and last seen dates. The only user data sent are values of the
+                            properties selected here.
+                        </p>
+                        <PropertySelect
+                            taxonomicFilterGroup={TaxonomicFilterGroupType.PersonProperties}
+                            sortable={false}
+                            onChange={(properties) => {
+                                updateSummaryConfig({
+                                    ...currentConfig,
+                                    important_user_properties: properties,
+                                })
+                            }}
+                            selectedProperties={currentConfig.important_user_properties || []}
+                            addText="Add property"
+                        />
+                    </div>
+                </>
+            )}
+        </div>
+    )
+}
+
+export function ReplayGeneral(): JSX.Element {
+    const { updateCurrentTeam } = useActions(teamLogic)
+    const { currentTeam } = useValues(teamLogic)
+
+    return (
+        <div className="flex flex-col gap-4">
+            <div>
+                <p>
+                    Watch recordings of how users interact with your web app to see what can be improved.{' '}
+                    <Link
+                        to="https://posthog.com/docs/user-guides/recordings?utm_campaign=session-recording&utm_medium=in-product"
+                        target="_blank"
+                    >
+                        Check out our docs
+                    </Link>
+                </p>
+                <LemonSwitch
+                    data-attr="opt-in-session-recording-switch"
+                    onChange={(checked) => {
+                        updateCurrentTeam({
```

Comment:
Should we be doing `ai_config: { ...ai_config, opt_in: false }`?

Reply:
ah, this is an artefact of me going backwards and forwards on how deeply we should support safe updates (like with the top level keys)

i really couldn't make my mind up - what do you think @daibhin ?

Reply:
so, I think we should go one level deep into this config, since without it those items would be at the top level

my thinking is that since these updates are all implemented as `PATCH`es it feels natural that

`{ src: { ai_config: {} }`

wouldn't affect 

`{ src: { replay_config: {}}`

that's also then a pressure on us not to deeply nest these settings

Reply:
Yep agreed, one level seems fair 👍